### PR TITLE
Turkish translations

### PR DESCRIPTION
**Pull request by Kara-Kartallar**
Turkish translations were made.

### CODE REVIEW

File: fun_with_flags/translations/tr/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -397,11 +418,15 @@ msgid ""
 "          You're in the\n"
 "          unicorn club. <br><i>Received"
 msgstr ""
+"\n"
+"        <b>Kulüp:</b>\n"
+"          Sen içindesin\n"
+"          unicorn kulübü. <br><i>Alındı</i>"
```
Comment (joschobart) [inline fun_with_flags/translations/tr/LC_MESSAGES/messages.po:424]: you shouldn't close the i tag here (</i>) it's closed later in the template. this is true for all similar for all three badges.

**Review by joschobart — CHANGES_REQUESTED** [review]

File: fun_with_flags/translations/tr/LC_MESSAGES/messages.po (hunk 1 above)
Comment (Kara-Kartallar) [reply fun_with_flags/translations/tr/LC_MESSAGES/messages.po]: fixed

Comment (joschobart) [reply fun_with_flags/translations/tr/LC_MESSAGES/messages.po]: cool, but there's the same issue sstill on lines 441 & 457. please also fix there. cheers!

Comment (Kara-Kartallar) [reply fun_with_flags/translations/tr/LC_MESSAGES/messages.po]: fixed again.

**Pull request merged**